src/workflows-app/SubmitWorkflow.js

```diff
@@ -126,51 +127,55 @@ export const SubmitWorkflow = wrapWorkflowsPage({ name: 'SubmitWorkflow' })(
       }
     });
 
-    return loading
-      ? centeredSpinner()
-      : div([
-          !pageReady && h(WorkflowsAppLauncherCard, { onClick: createWorkflowsApp, disabled: launcherDisabled }),
-          pageReady &&
-            div({ style: { margin: '2rem 4rem' } }, [
```

Comment:
to make this diff more obvious: I moved this `div` (the main contents of the SubmitWorkflow page) into its own function, `renderSubmitWorkflow` (see added code below), and moved the flow control to a `Utils.cond` statement.